vitest.shared.mjs

```diff
@@ -1,4 +1,5 @@
 import inspector from 'node:inspector';
+import { URL } from 'node:url';
```

Comment:
Not that it matters much, but isn't it also a global?

Reply:
eslint complains here 🤔